storage/innobase/xtrabackup/test/t/pxb-2797.sh

```diff
@@ -0,0 +1,34 @@
+#
+# PXB-2797: Schema mismatch when importing table with full-text index from backup
+#
+
+start_server
```

Comment:
Can you extend the test case and make sure other types of index that can have ASC or DESC are working properly?

Reply:
ack

Reply:
the only other index type that can have DESC is Hash index, but it is not supported by PXB storage engines. So I think no need to test this case

Reply:
One nit picking I missed from the last review. Can we rename the test case to something more meaningful? We are trying to avoid just plain JIRA issue as the name. Something like pxb-2797-export_fulltext.sh will do it.